pavelib/assets.py

```diff
@@ -172,6 +196,7 @@ def compile_templated_sass(systems, settings):
     """
     for sys in systems:
         sh(django_cmd(sys, settings, 'preprocess_assets'))
+        print("\t\tFinished preprocessing assets.")
```

Comment:
IIRC one of them (cms/lms) takes way longer than the other. Or even if not, I'd prefer something like:
```
print("\t\tFinished preprocessing {} assets.".format(sys))
```